templates/caching-service.json

```diff
@@ -292,7 +292,6 @@
                               "memory": "${TOTAL_CONTAINER_MEM}Mi"
                            },
                            "limits": {
-                              "cpu": "0.5",
```

Comment:
Nitpick, but should we have these as a separate commit so it's more obvious in the history that this change has been made. I don't associate testing scaling with removing cpu limits from the template :smile:

Reply:
You're absolutely right! The point of this change is to let Infinispan go faster if there's free CPU on the machine. The downside is that we are degraded from the "Guaranteed QoS" to "Burstabe QoS". So I have a mixed feelings about it. On one hand why we shouldn't go faster if we can but on the other hand we can be killed if there's not sufficient CPU on the node.... Taught choice...